docs/rules/no-classic-classes.md

```diff
@@ -38,13 +38,19 @@ import Evented from '@ember/object/evented';
 export default class MyComponent extends Component.extend(Evented) {}
 ```
 
+## Configuration
+
+This rule takes an optional object containing:
+
+- `array` -- `additionalInvalidImports` -- Allows you to specify additional imports that should be flagged to disallow calling `extend` on. This allows you to handle the case where your app or addon is importing from a module that performs the `extend`.
```

Comment:
Thoughts about `additionalClassImports` as the name to make this more specific about what imports it's talking about?

Reply:
Ya I like.